app/pages/Covid/Covid.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Footer from '../../components/ui/Footer/Footer';
+import Iframe from 'react-iframe';
```

Comment:
I'm not familiar with this library. Is there a reason we need it here? Was the regular iframe not working?

Reply:
It allows us to easily strip out the CSS of a google chrome embed and allow it to be more readable. There may be some better workarounds but I wanted to get this out sooner.

Reply:
ah ok cool, sounds good!